aloscene/io/depth.py

```diff
@@ -8,12 +8,14 @@ def load_depth(path):
     Parameters
     ----------
     path: str
-        path to the disparity file. Supported format: {".npy"}. If your file is stored differently, as an
+        path to the disparity file. Supported format: {".npy",".npz"}. If your file is stored differently, as an
         alternative, you can open the file yourself and then create the Depth augmented Tensor from the depth
         data.
     """
     if path.endswith(".npy"):
         return np.load(path)
+    elif path.endswith(".npz"):
+        return list(np.load(path).values())
```

Comment:
Not sure we want to return the list of delement here. I think we want to retrun only one depth element assuming only one depth is inside the npz. Somethine link
```
content = np.load(path)
return content[content.keys()[0]]
```